dev-packages/ovsx-client/src/ovsx-client.ts

```diff
@@ -160,7 +167,7 @@ export class OVSXClient {
      */
     getLatestCompatibleVersion(entry: VSXSearchEntry): VSXAllVersions | undefined {
         const extensions = entry.allVersions;
-        if (this.isBuiltinNamespace(entry.namespace)) {
+        if (this.isBuiltinExtension(`${entry.namespace}.${entry.name}`)) {
```

Comment:
In the initial change, I did considered this name but I thought it would be confusing,
as at a first impression the reader may think it's referring to the builtin extensions of the specific application being handled.  but this is more a type of extension.
may be we can consider `isBuiltinType`?

Reply:
We never used the terminology `builtin type` before so it might add more confusion.
I've also made sure to properly document the method to reference what it is used for.

The check is to determine if a vscode-managed builtin is present (vscode and eclipse-theia namespace, with external vscode builtins). If you think it might be more descriptive, I could rename the method to:

```ts
protected isVscodeBuiltinExtension(id: string): boolean {
```